src/__tests__/ChildReducer.test.js

```diff
@@ -17,6 +17,7 @@ describe('childReducer test suite', () => {
     losses: null,
     achievements: null,
     Ballots: [],
+    Streaks: 'test',
```

Comment:
Team B's recent merge caused the ChildReducer test to fail. The initial state in the test file was missing the 'Streaks' property.

Reply:
Nice catch